test/unit/offset.js

```diff
@@ -57,12 +57,11 @@ test("object without getBoundingClientRect", function() {
 });
 
 test("disconnected node", function() {
-	expect(2);
+	expect(1);
 
 	var result = jQuery( document.createElement("div") ).offset();
 
-	equal( result.top, 0, "Check top" );
-	equal( result.left, 0, "Check left" );
+	equal( typeof result, "undefined", "Check result typeof" );
```

Comment:
A bit too precise, especially given that the call will throw on IE8 in the cherry-pick to `compat`... how about a try/catch and `ok( !result, "no position for disconnected element" )`?

And we'll want the same for a `display: none` element.